website/source/docs/agent/http/kv.html.markdown

```diff
@@ -142,3 +149,228 @@ DELETE request:
+    <td align="center">X</td>
+    <td align="center">O</td>
+    <td align="center"></td>
+    <td align="center">X</td>
+  </tr>
+  <tr>
+    <td>get</td>
+    <td>Gets the `Key` during the transaction. This fails the transaction if the `Key` doesn't exist. The key may not be present in the results if ACLs do not permit it to be read.</td>
+    <td align="center">X</td>
+    <td align="center"></td>
+    <td align="center"></td>
+    <td align="center"></td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>get-tree</td>
+    <td>Gets all keys with a prefix of `Key` during the transaction. This does not fail the transaction if the `Key` doesn't exist. Not all keys may be present in the results if ACLs do not permit them to be read.</td>
+    <td align="center">X</td>
+    <td align="center"></td>
+    <td align="center"></td>
+    <td align="center"></td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>check-index</td>
+    <td>Fails the transaction if `Key` does not have a modify index equal to `Index`.</td>
+    <td align="center">X</td>
+    <td align="center"></td>
+    <td align="center"></td>
+    <td align="center">X</td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>check-session</td>
+    <td>Fails the transaction if `Key` is not currently locked by `Session`.</td>
+    <td align="center">X</td>
+    <td align="center"></td>
+    <td align="center"></td>
+    <td align="center"></td>
+    <td align="center">X</td>
+  </tr>
+  <tr>
+    <td>delete</td>
+    <td>Deletes the `Key`.</td>
+    <td align="center">X</td>
+    <td align="center"></td>
+    <td align="center"></td>
+    <td align="center"></td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>delete-tree</td>
+    <td>Deletes all keys with a prefix of`Key`.</td>
+    <td align="center">X</td>
+    <td align="center"></td>
+    <td align="center"></td>
+    <td align="center"></td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>delete-cas</td>
+    <td>Deletes the `Key` with check-and-set semantics. The `Key` will only be deleted if its current modify index matches the supplied `Index`.</td>
+    <td align="center">X</td>
+    <td align="center"></td>
+    <td align="center"></td>
+    <td align="center">X</td>
+    <td align="center"></td>
+  </tr>
+</table>
+
+If the transaction can be processed, a status code of 200 will be returned if it
+was successfully applied, or a status code of 409 will be returned if it was rolled
+back. If either of these status codes are returned, the response will look like this:
+
+```javascript
+{
+  "Results": [
+    {
+      "KV": {
+        "LockIndex": <lock index>,
+        "Key": "<key>",
+        "Flags": <flags>,
+        "Value": "<Base64-encoded blob of data, or null>",
+        "CreateIndex": <index>,
+        "ModifyIndex": <index>
+      }
+    },
+    ...
+  ],
+  "Errors": [
+    {
+      "OpIndex": <index of failed operation>,
+      "What": "<error message for failed operation>"
```

Comment:
Ah now I remember why I did that - if you have a member named `Error` you can't have the `Error()` method on it for that interface.